internal/rpc/block_write_stream.go

```diff
@@ -341,22 +341,14 @@ func (s *blockWriteStream) writePacket(p outboundPacket) error {
 	binary.BigEndian.PutUint32(header, uint32(totalLength))
 	binary.BigEndian.PutUint16(header[4:], uint16(len(infoBytes)))
 	header = append(header, infoBytes...)
+	header = append(header, p.checksums...)
+	header = append(header, p.data...)
```

Comment:
Can you give `header` some extra capacity up on line 333?

Reply:
done!